RemoteWellnessSupportApp/Activity/PhysicalCondition/ViewModels/PhysicalConditionEntryFormViewModel.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  PhysicalConditionEntryFormViewModel.swift
+//  RemoteWellnessSupportApp
+//
+//  Created by 岩本雄貴 on 2024/02/17.
+//
+
+import Combine
+import Foundation
+import SwiftData
+
+class PhysicalConditionEntryFormViewModel: ObservableObject {
+    @Published var selectedDateTime = Date()
+    @Published var memo = ""
+    @Published var selectedRating: PhysicalConditionRating?
+    @Published var isErrorAlert: Bool = false
```

Comment:
[nits]
細かいですが、変数に初期値を指定した場合は、型推論が効くので、型宣言は省略することが多いです。

```suggestion
    @Published var isErrorAlert = false
    @Published var errorMessage = ""    
```

Reply:
毎度すいません
下記にて修正しました
https://github.com/y-iwamoto/RemoteWellnessSupportApp/pull/6/commits/9c9317234c224bcf65832692064cbbe38805819b